### PR TITLE
mkcloud: add develcloud7/susecloud7

### PR DESCRIPTION
**Pull request by dirkmueller**
Also remove susecloud6 (no longer supported).

### CODE REVIEW

**Comment by toabctl** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4158,7 +4167,7 @@ function onadmin_run_cct()
             develcloud5|GM5|GM5+up)
                 checkout_branch=cloud5
                 ;;
-            develcloud6)
+            develcloud6|develcloud7)
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: could we use here iscloudver 6plus ?

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -416,8 +416,10 @@ function getcloudver()
         echo -n 4
     elif [[ $cloudsource =~ ^.*(cloud|GM)5(\+up)?$ ]] ; then
         echo -n 5
-    elif [[ $cloudsource =~ ^(.+6|M[[:digit:]]+|Beta[[:digit:]]+|RC[[:digit:]]*|GMC[[:digit:]]*|GM6?(\+up)?)$ ]] ; then
+    elif [[ $cloudsource =~ ^.*(cloud|GM)6(\+up)?$ ]] ; then
         echo -n 6
+    elif [[ $cloudsource =~ ^(.+7|M[[:digit:]]+|Beta[[:digit:]]+|RC[[:digit:]]*|GMC[[:digit:]]*|GM7?(\+up)?)$ ]] ; then
+        echo -n 7
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: this means that we cannot install any older cloud5 Milestones anymore, but if that is fine with you, still +1

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: cloud5? you mean cloud6 ? yeah, would be fine by me.

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: we only want to run it in the develcloud case not in the susecloud case (I forgot why)

**Pull request merged**